deploy/manifests/nginx-gateway.yaml

```diff
@@ -124,8 +124,8 @@ spec:
         - static-mode
         - --gateway-ctlr-name=k8s-gateway.nginx.org/nginx-gateway-controller
         - --gatewayclass=nginx
-      - image: nginx:1.23
-        imagePullPolicy: IfNotPresent
+      - image: nginx:1.25
+        imagePullPolicy: Always
```

Comment:
`IfNotPresent` is generally the better option, but if we want to keep that as the default, we'd probably need to rephrase our docs in regards to pulling the latest patch version, since it only happens on the first pull of the image.

Reply:
I'm pretty sure `Always` just means that it will check if there's a new image, but if the same version already exists it won't actually pull anything (that's what Docker does too). Not sure if Docker Hub counts that in their pull limit though.

I think it would be better to add the patch version here imo (a lot of projects also point to the SHA), but that still won't solve the problem with `IfNotPresent` that it will never pull a newer version of the image for the same tag, like when the image gets re-built for CVEs.